tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -112,7 +112,13 @@ def get_function_app_params(self):
                 'location': 'eastus',
                 'resource_group_name': 'cloud-custodian',
                 'sku_tier': 'Dynamic',  # consumption plan
-                'sku_name': 'Y1'
+                'sku_name': 'Y1',
+                'auto_scale':{
+                    'enable_auto_scale': False,
+                    'min_capacity':1,
```

Comment:
I think these parameters should be None when enable_auto_scale is False.

Reply:
The API requires values for all 4 fields. If one wants to change auto scale to false, one still has to pass in valid values for min, max, and default

Reply:
We don't make any changes with existing resources to avoid accidental user configuration overrides, so it should not be an issue. If AppServicePlan already exists, we should skip autoscale configuration (please see another comment)